spotbugs/src/main/java/edu/umd/cs/findbugs/Version.java

```diff
@@ -20,28 +20,34 @@
 package edu.umd.cs.findbugs;
 
 import java.io.InputStream;
+import java.lang.invoke.MethodHandles;
 import java.net.URL;
 import java.util.Enumeration;
 import java.util.jar.Manifest;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import javax.annotation.CheckForNull;
 
 /**
  * Version number and release date information.
  */
 public class Version {
+
+    private static final Logger LOGGER = Logger.getLogger(MethodHandles.lookup().lookupClass().getName());
```

Comment:
I personally prefer just LOG.

Reply:
Why not just Version.class.getName()?

Reply:
LOG or LOGGER works. But the class name is Logger, so...eh.

The reason for using `MethodHandles` is so that this line can be copy-pasted to any other class without modification.